Export/AbstractExport.php

```diff
@@ -37,15 +39,19 @@ abstract class AbstractExport implements \JsonSerializable
     protected $latestRow = 0;
 
     /**
-     * Constructor.
+     * AbstractExport constructor.
+     *
+     * @param CacheItemPoolInterface $cachePhpspreadsheet
      *
      * @throws \Exception
      */
-    public function __construct()
+    public function __construct(CacheItemPoolInterface $cachePhpspreadsheet)
     {
         if (empty($this->title)) {
             throw new \Exception('Export title not defined.');
         }
+
+        \PhpOffice\PhpSpreadsheet\Settings::setCache(new Psr16Cache($cachePhpspreadsheet));
```

Comment:
Any specific reason for not using `use PhpOffice\PhpSpreadsheet\Settings;`?

Reply:
Done in 3eea330